_posts/2018-06-18-configuration.md

```diff
@@ -0,0 +1,160 @@
+---
+author: Sean Callan
+author_link: https://github.com/doomspork
+categories: general
```

Comment:
Should this be classified under something other than `general`?  What categories should we have?

Reply:
We're not using the `categories` feature yet (or `tags`). At a later date we could perhaps use it to group similar posts?